### DISCUSSION TITLE
how to save electron log on file name with each date?

### DISCUSSION THREAD

**Discussion by debchy** [megahertz/electron-log#239] [Q&A]
Hi,
I wanted to save  logs of everyday into log file of that day. Is there any way for that?

**Comment by megahertz** [comment]
You can set a custom resolvePath option of file transport.

**Reply by kyiv-sdk** [reply]
@megahertz does it have an option to log only say 20MB and then create a new log file?

**Reply by megahertz** [reply]
https://github.com/megahertz/electron-log/blob/master/docs/file.md#maxsize-number ?
